infor-design/enterprise
# 329 - Disabled searchfield placeholders

**Explain the _details_ for making this change. What existing problem does the pull request solve?**
- Disabled the "No Results" and "All Results" placeholders
- http://localhost:4000/components/searchfield/example-clearable.html

**Related github/jira issue (required)**:
Closes #329 

**Steps necessary to review your pull request (required)**:
- Type "test" in the search field
- Click "No Results", "No Results" should be disabled
- Clear the value of the search field then type "al"
- Click "All Results For "al", "All Results For "al" should be disabled
